crates/burn-core/src/lib.rs

```diff
@@ -62,3 +62,18 @@ pub type TestAutodiffBackend = burn_autodiff::Autodiff<TestBackend>;
 /// LearningRate also implements [learning rate scheduler](crate::lr_scheduler::LrScheduler) so it
 /// can be used for constant learning rate.
 pub type LearningRate = f64; // We could potentially change the type.
+
+pub mod prelude {
+    //! Structs and macros used by most projects. Add `use
+    //! burn::prelude::*` to your code to quickly get started with
+    //! Burn.
+    pub use crate::{
+        config::Config,
+        module::Module,
+        nn::{loss::*, *},
+        tensor::{
+            backend::{AutodiffBackend, Backend},
+            Data, Tensor,
+        },
```

Comment:
I think this would be better:

```rust
pub use crate::{
    config::Config,
    module::Module,
    nn,
    tensor::{
        backend::Backend,
        Data, Tensor, ElementConversion, Device
    },
};
```

I wouldn't include any autodiff-related items for now, and I believe the `nn` module is sufficient.

Reply:
Yeah, I agree with it! Users should also take into account the usage of autodiff-related items as the `train` feature can be enabled when necessary now.